docs/build_mozc_in_windows.md

```diff
@@ -15,8 +15,12 @@ cd mozc\src
 
 python build_tools/update_deps.py
 python build_tools/build_qt.py --release --confirm_license
+# Choose one of the following:
+# 1. Build with GYP
 python build_mozc.py gyp
 python build_mozc.py build -c Release package
+# 2. Build with Bazel
```

Comment:
Since this is a summary of typical build steps, I'd like to keep it simple. 

How about writing a summary with Bazel in the Bazel section instead?

The path to `Mozc64.msi` is also different between GYP and Bazel.